base/loading.jl

```diff
@@ -1594,23 +1596,26 @@ end
 Helper function to merge preference dicts recursively, honoring overrides in nested
 dictionaries properly.
 """
-function recursive_prefs_merge(base::Dict, overrides::Dict...)
+function recursive_prefs_merge(base::Dict{String, Any}, overrides::Dict{String, Any}...)
     new_base = Base._typeddict(base, overrides...)
 
     for override in overrides
+        override isa Dict{String, Any} || continue # error here?
```

Comment:
You've constrained `overrides` already, I don't think this check is necessary.

Reply:
Good point, agree.